src/main/cli/cli.c

```diff
@@ -5780,6 +5792,7 @@ const clicmd_t cmdTable[] = {
 #ifdef USE_FLASH_TOOLS
     CLI_COMMAND_DEF("flash_read", NULL, "<length> <address>", cliFlashRead),
     CLI_COMMAND_DEF("flash_write", NULL, "<address> <message>", cliFlashWrite),
+    CLI_COMMAND_DEF("flash_verify", NULL, "erase, write and verify flash", cliFlashVerify),
```

Comment:
That's not very alphabetical. :wink: Also, the second parameter is the description, third is command parameters.